db/migrate/20200530172450_delete_disable_account_registration_from_settings.rb

```diff
@@ -0,0 +1,11 @@
+class DeleteDisableAccountRegistrationFromSettings < ActiveRecord::Migration[5.2]
+  def self.up
+    return unless HackathonConfig.find_by(var: 'disable_account_registration').present?
+    HackathonConfig.find_by(var: 'disable_account_registration').destroy
+  end
+
+  def self.down
+    return unless HackathonConfig.find_by(var: 'disable_account_registration').nil?
+    HackathonConfig.create(var: 'disable_account_registration', value: false).save
```

Comment:
Wouldn't this not restore any previously set value? Or does it not matter?

Reply:
Shouldn't matter and we'd have to hang on to the old value somewhere.